src/engine/qcommon/translation.cpp

```diff
@@ -281,7 +281,7 @@ void Trans_LoadDefaultLanguage()
 		FL_FindLocale( &locale, FL_MESSAGES );
 
 		// Invalid or not found. Just use builtin language.
-		if( !locale->lang || !locale->lang[0] || !locale->country || !locale->country[0] )
```

Comment:
What is the PVS studio finding here?

Reply:
In case `country` is not present in locale info, we reverted locale to "en". But in `else` branch we see a fallback when `country` is not present - use just `lang` member to choose game language.

So was decided to remove `country` check above and just use `lang` for game language selection, if country is not present.